exercises/bracket-push/src/example.c

```diff
@@ -0,0 +1,50 @@
+#include "bracket_push.h"
+#include <stdio.h>
+
+static bool is_bbp(char c)
```

Comment:
`bbp` is bracket, brace, or parenthesis?

Reply:
Yes

Reply:
might be better to make this `is_brace_bracket_paren()` or maybe split it into `is_brace()`, `is_bracket()` & `is_parenthesis()` (if going the latter route you could possibly add `inline` ).